build/build.xml

```diff
@@ -616,6 +621,7 @@
 				<include name="selenium-${status}-*.zap"/>
 				<include name="spiderAjax-${status}-*.zap"/>
 				<include name="websocket-${status}-*.zap"/>
+				<include name="fieldenumeration-${status}-*.zap"/>
```

Comment:
Including the addon/extension here means that it'll be included in weekly releases, per line 605.